generators/ci-cd/command.ts

```diff
@@ -73,113 +84,147 @@ const command: JHipsterCommandDefinition = {
           value: 'cypressDashboard',
         },
       ],
-      scope: 'context',
+      scope: 'storage',
     },
     insideDocker: {
-      prompt: {
-        when: includesValue('ciCd', ['jenkins', 'gitlab']),
+      cli: {
+        type: Boolean,
+      },
+      prompt: () => ({
+        when: answers => answers.ciCd?.includes('jenkins') || answers.ciCd?.includes('gitlab'),
         type: 'confirm',
         message: 'Would you like to perform the build in a Docker container ?',
         default: false,
-      },
-      scope: 'context',
+      }),
+      scope: 'storage',
     },
     sendBuildToGitlab: {
-      prompt: {
-        when: includesValue('ciCd', ['jenkins']),
+      cli: {
+        type: Boolean,
+      },
+      prompt: () => ({
+        when: answers => answers.ciCd?.includes('jenkins'),
         type: 'confirm',
         message: 'Would you like to send build status to GitLab ?',
         default: false,
-      },
-      scope: 'context',
+      }),
+      scope: 'storage',
     },
     artifactorySnapshotsId: {
-      prompt: {
-        when: includesValue('ciCdIntegrations', ['deploy']),
+      cli: {
+        type: String,
+      },
+      prompt: () => ({
+        when: answers => answers.ciCdIntegrations?.includes('deploy'),
         type: 'input',
         message: `${chalk.yellow('*Artifactory*')}: what is the ID of distributionManagement for snapshots ?`,
-      },
-      default: 'snapshots',
-      scope: 'context',
+        default: 'snapshots',
```

Comment:
Is there a formatting issue here?

Reply:
Don't think so: it renders properly in edit mode